package.json

```diff
@@ -43,7 +43,7 @@
     "sinon-chai": "^3.3.0"
   },
   "dependencies": {
-    "@dashevo/dpp": "^0.9.0",
+    "@dashevo/dpp": "github:dashevo/js-dpp#identity-create-st",
```

Comment:
Well I guess we have to release dpp with identities before this can be merged

Reply:
Meanwhile, we can use `dev` channel.